external-crates/move/crates/move-compiler/src/expansion/primitive_definers.rs

```diff
@@ -34,14 +34,16 @@ pub fn modules(
         )
     }
     if let Some(pre_compiled_lib) = pre_compiled_lib_opt {
-        for (mident, m) in pre_compiled_lib.expansion.modules.key_cloned_iter() {
-            check_prim_definer(
-                &reporter,
-                /* allow shadowing */ true,
-                &mut definers,
-                mident,
-                m,
-            )
+        for pre_compiled_lib in pre_compiled_lib {
+            for (mident, m) in pre_compiled_lib.expansion.modules.key_cloned_iter() {
+                check_prim_definer(
+                    &reporter,
+                    /* allow shadowing */ true,
+                    &mut definers,
+                    mident,
+                    m,
+                )
```

Comment:
What happens when the compiler is invoked with multiple versions of a package at the same name, down the dependency chain?